sql/20190428223823_create_loans_table.sql

```diff
@@ -0,0 +1,23 @@
+-- migrate:up
+CREATE TABLE "loans" (
+    "id"                   SERIAL  NOT NULL,
+    "client_id"            INT     NOT NULL,
+    "application_id"       INT     NOT NULL,
+    "amount"               INT     NOT NULL,
+    "term"                 INT     NOT NULL,
+    "interest"             INT     NOT NULL,
+    "currency"             TEXT    NOT NULL,
+    "date"                 TEXT    NOT NULL,
+    "completed"            BOOLEAN NOT NULL,
```

Comment:
finished ?

Reply:
We discussed it in the specification and "completed" was fine, also in loans, we use "completed"

Reply:
okey